projects/centeva-core/src/components/table/table.component.ts

```diff
@@ -60,9 +69,21 @@ export class TableComponent implements OnInit {
 
   async ngOnInit() { }
 
+  ngOnDestroy() {
+    for (const key in this.checkboxModels) {
```

Comment:
hmm I wouldn't expect to see logic happening in an onDestroy. Is there maybe a better way to get these checkboxEvents that doesn't depend on the component lifecycle?